lib/rules/web/admin_console/4.4/common_ui_elements.xyaml

```diff
@@ -597,10 +597,6 @@ check_restricted_access_text:
   params:
     content: Restricted Access
   action: check_page_contains
-check_no_configmaps_found_text:
-  params:
-    content: No Config Maps Found
-  action: check_page_contains
```

Comment:
`check_no_configmaps_found_text` is no longer needed after cucushift#8345 is merged so removed it